scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -470,7 +525,7 @@ main (int argc, char *argv[])
 
   /** Connect Traces **/
   AsciiTraceHelper ascii;
-  Ptr<OutputStreamWrapper> outputSlsPhase = ascii.CreateFileStream ("slsResults" + arrayConfig + ".csv");
+  Ptr<OutputStreamWrapper> outputSlsPhase = ascii.CreateFileStream ("../output/slsResults" + arrayConfig + ".csv");
```

Comment:
the folder where the code put its output files is related to users preferences.. I'd avoid specifying folders on public code, I'd do it just on my local repo :) It could create less issues also on possible merges with the official repo